website/docs/migration/from_state_notifier/build_init_old/index.tsx

```diff
@@ -0,0 +1,7 @@
+import raw from "!!raw-loader!./raw.dart";
```

Comment:
In folders with nothing but an index + raw.dart, could you remove the folder and rename `raw.dart to `<folder_name>.dart`?

Then have the mdx doc import `!!raw-loader/doc_name/<folder_name>.dart`

No need to create tons of files for nothing

Reply:
You can then update `AutoSnippet` to `<AutoSnippet raw={snippet} />`

Use the `{...snippet}` only when there's actual generated code.

Reply:
Thanks ❤️